internal/plugincommon/consensus/threshold.go

```diff
@@ -0,0 +1,47 @@
+package consensus
+
+// Threshold is the minimum number of observations required for a given chain.
+type Threshold uint
+
+// MultiThreshold is an interface that maps a chain selector to a threshold.
+type MultiThreshold[T any] interface {
+	// Get the minimum number of observations required for a given chain.
+	Get(chain T) (Threshold, bool)
+}
+
+// TwoFPlus1 is a common threshold mapping function that returns 2*f + 1.
+func TwoFPlus1(f int) Threshold {
+	return Threshold(2*f + 1)
+}
+
+// FPlus1 is a common threshold mapping function that returns f + 1.
+func FPlus1(f int) Threshold {
```

Comment:
libocr describes FPlus1 as "Guarantees at least one honest observation", can we put that in the comment?